proto_v2/instr_v2.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package fd.v2;
+
+/* As a note, the accounts and account data fields are populated at a higher
+   level than the instruction. */
+message InstrEnv {
+    /* Index into the set of txn accounts defined by the TxnEnv. */
+    uint32          program_id_index = 1;
+
+    /* Indexes into the set of txn accounts defined by the TxnEnv. */
+    repeated uint32 accounts         = 2;
```

Comment:
Preserve the `InstrAcct` field since we need to include writable and signer status for instructions